android/library/src/main/java/com/liferay/mobile/screens/context/AuthenticationType.java

```diff
@@ -19,5 +19,21 @@
  */
 public enum AuthenticationType {
 
-	BASIC, OAUTH, VOID
+	BASIC(0), OAUTH(1), COOKIE(2), VOID(3);
+
+	private final int value;
+
+	AuthenticationType(int value) {
+		this.value = value;
+	}
+
+	public static AuthenticationType getValue(int value) {
```

Comment:
I think enums can already do this automatically or at least AuthenticationType.values()[value]